packages/cms-editor/src/components/CmsValueField.test.tsx

```diff
@@ -28,7 +28,10 @@ test('readonly', () => {
 });
 
 const renderCmsValueField = (values: MapStringString, readonly?: boolean) => {
-  const ui = (values: MapStringString) => <CmsValueField values={values} updateValue={() => {}} deleteValue={() => {}} languageTag='en' />;
+  const ui = (values: MapStringString) => (
+    // eslint-disable-next-line i18next/no-literal-string
```

Comment:
hmm... test files should be ignroed by this lint rule?
![image](https://github.com/user-attachments/assets/3b3cb43f-b8c5-4b2e-8f1f-f8777295d201)